### ISSUE TITLE
Package load error

### ISSUE THREAD

**Issue by jaliasd** [ICPI/ICPIutilities#17]
> Error: package or namespace load failed for 'ICPIutilities' in library.dynam(lib, package, package.lib):
>  DLL 'rlang' not found: maybe not installed for this architecture?

**Comment by achafetz** [comment]
Unsure why rlang is required but it must be a dependent of another package that's being used (although why is it required only now?). Will add `rlang` into the dependencies to have it installed with this package.

**Comment by achafetz** [comment]
reopening issue. @abrahamagedewICPI & @kimisatoPC still experiencing even with rlang included as an import. Current work around for now is to install rlang and then update/install ICPIutilities --> `install.packages("rlang")`

https://github.com/ICPI/ICPIutilities/blob/6069ee20de10b70540d131cec30be4981044e845/DESCRIPTION#L8-L16

**Comment by achafetz** [comment]
@jason-p-pickering, question 2 of the day.

ICPIutiltities started requiring rlang maybe a week and a half ago. I looked at the packages in my [Imports](https://github.com/ICPI/ICPIutilities/blob/6069ee20de10b70540d131cec30be4981044e845/DESCRIPTION#L8-L16) to see what package needed it and it looks like dplyr and tidyr now have rlang as a dependency. 

```
pkgs <- c("data.table","dplyr","knitr","magrittr","readr","readxl","rlang","stringr", "tidyr")
purrr::map(.x = pkgs, .f = ~ tools::package_dependencies(.x)) %>% purrr::flatten()
```
Josh noted the issue last week and I added rlang to the Imports assuming this would fix the issue, but it appears it hasn't.  Folks are still getting the below error message.

> Error: package or namespace load failed for 'ICPIutilities' in library.dynam(lib, package, package.lib): DLL 'rlang' not found: maybe not installed for this architecture?

They can install rlang prior to installing ICPIutilities, but I assumed that if it was in the Imports section of the Description file that it would be installed if it was missing. I looked it up in [Hadley's _R Packages_](http://r-pkgs.had.co.nz/description.html#dependencies) and it reads like this should be the case.

> Imports: packages listed here must be present for your package to work. In fact, any time your package is installed, those packages will, if not already present, be installed on your computer (devtools::load_all() also checks that the packages are installed).

Any thoughts on why this is causing an issue for people installing/updating the package?
